api/src/main/java/uk/gov/dvsa/mot/trade/service/DvlaVehicleFirstMotDueDateCalculator.java

```diff
@@ -0,0 +1,202 @@
+package uk.gov.dvsa.mot.trade.service;
+
+import uk.gov.dvsa.mot.persist.model.DvlaVehicle;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
I'm getting an error with these imports. Are they IDE specific imports? If so, we shouldn't couple our app to the IDE.